README.md

```diff
@@ -113,13 +114,23 @@ rails server
 
```

Comment:
- [ ] The new expense button on the category show page should be placed correctly.

  ![image](https://github.com/M-Anwar-Hussaini/Expense-Management/assets/36542986/79cd3bd9-93ff-4e08-a240-e64e7329c303)